oauth2_http/java/com/google/auth/oauth2/OAuth2Utils.java

```diff
@@ -70,11 +70,15 @@
 import java.util.Set;
 
 /** Internal utilities for the com.google.auth.oauth2 namespace. */
-class OAuth2Utils {
+public class OAuth2Utils {
```

Comment:
Not sure if it makes sense for this to be public @lqiu96

Reply:
One option for this to see if we can keep this package-private and move some of these constants to STSRequestHandler. It looks like `TOKEN_TYPE_ACCESS_BOUNDARY_INTERMEDIARY_TOKEN` and `TOKEN_EXCHANGE_URL_FORMAT` are new additions could be live in the individual classes (i.e StsREquestHandler or DownscopedCredentials).

TOKEN_TYPE_ACCESS_TOKEN is used in a few places, but if it makes sense we can move them around. I don't know if `TOKEN_TYPE_ACCESS_TOKEN` applies to non-CAB use cases.

Otherwise, I think I'm ok with making it public given the module constraints we have.

Reply:
I'm going to leave this comment open and keep it as-is for now so I can merge this PR and unblock #1571. I'll address this comment in the next PR.